detect.go

```diff
@@ -40,6 +43,11 @@ func Detect(logger scribe.Emitter) packit.DetectFunc {
 }
 
 func WebRoot(workingDir string) (string, error) {
```

Comment:
I know this isn't part of this PR but I see that we never use the return string value. Maybe there is a plan to consume it somewhere in the future?

Reply:
we use it in the [other buildpack](https://github.com/ninech/buildpack-static-confgen/blob/main/detect.go#L29) :)